lib/matplotlib/_api/__init__.py

```diff
@@ -151,12 +151,10 @@ def check_shape(shape, /, **kwargs):
         if (len(data_shape) != len(shape)
                 or any(s != t and t is not None for s, t in zip(data_shape, shape))):
             dim_labels = iter(itertools.chain(
-                'MNLIJKLH',
+                'NMLIJKLH',
```

Comment:
It's not new, but should L appear in this list twice? That seems odd/likely for unintended

Reply:
indeed, fixed.